src/blake3_stubs.c

```diff
@@ -9,7 +9,12 @@
 #include <caml/threads.h>
 #include <caml/unixsupport.h>
 
+#ifdef _WIN32
+#include <io.h>
+#define read _read
```

Comment:
How does this work in dune then? We use `read` there without any problems as far as I can tell

Reply:
MSVC gives a deprecation warning when it encounters `_read` but the OCaml compiler C libraries explicitly disable that warning so `read` works fine.

Here are the Microsoft docs:

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/posix-read?view=msvc-170

I'll get rid of the `#define` since it doesn't appear to affect us, I was just being cautious.

Reply:
I actually don't know why do we need unistd.h or io.h here at all. What happens when we remove these?